FogCreek/Glitch-Community
# Vivacious shirt: Fix /vscode-auth multi-prompt

## Links
* Remix link: https://vivacious-shirt.glitch.me/

## Changes:
* Before bouncing users to VS Code to complete the authentication process, navigate to `/` without a page refresh . This prevents the confirmation prompt from being re-triggered when the browser deals with going to a custom URI (`vscode://`) that is handled by the system by redirecting back.
* Also update some copy to reflect that signing in with email is no longer its own command in the extension.

## How To Test:
* Set a `persistentToken`
* Go to https://vivacious-shirt.glitch.me/vscode-auth
* Observe that dismissing whatever prompt appears does not result in another prompt

## Feedback I'm looking for:

Things I did wrong? (This is my first pull request on this project and the last time I wrote React code was 2017.)
